### DISCUSSION TITLE
Web Credentials

### DISCUSSION THREAD

**Discussion by WebShells** [opensourcepos/opensourcepos#3667] [Team Posts]
Hello Everybody,
Please comment with your emails in order to send you web credentials for support/ticketing portal.
Thank you!

**Comment by WebShells** [comment]
@daN4cat @SteveIreland @objecttothis

**Comment by objecttothis** [comment]
objecttothis@gmail.com.... Are we not going to be using Github issues tracker for support/ticketing any longer?

**Comment by WebShells** [comment]
Yes, we will still be relying on github issues as per the web also redirects to github threads.
The web credentials might be needed in case additional features were suggested by user who are willing to take next level of support and for contributers who are willing to take extra cash.
In addition, the ticketing portal will be more user friendly for users who finds us over google or other search engines than github, to what i believe...

**Comment by WebShells** [comment]
Please if anyone have different opinions maybe am wrong, but this is the first step for additional exposure and more users will be using it...

**Comment by WebShells** [comment]
@opensourcepos/org Please all confirm that you received an email from OSPOS with your accounts/password.
Thank you.

**Comment by SteveIreland** [comment]
Confirmed.  Thank ye kindly.

**Comment by objecttothis** [comment]
Received
